### ISSUE TITLE
mp3 file of the same song returns different alignment results

### ISSUE THREAD

**Issue by casillano** [lowerquality/gentle#276]
Tested on https://www.youtube.com/watch?v=B8WjnoZsXXM

I am using two methods to download files: the first is https://ytmp3.cc/en13/ and the second is https://www.npmjs.com/package/youtube-dl. When I convert a video using the first method and pass the audio into the aligner, it works as expected. But using the second method,  the aligner is not as accurate. The transcript used is the same for both methods.

I have also noticed that the file sizes for the two mp3 files differ (the mp3 from the first method is larger than the mp3 from the second method.

Note that for the first method, I am downloading the mp3 onto windows while on the second method I am downloading to linux.

**Comment by natelawrence** [comment]
In my experience, you can submit the exact same input media with identical transcript multiple times and get different results each time.

**Comment by casillano** [comment]
Right, they may vary a bit. But from what I've seen, converting and downloading videos from the site results in consistently better alignment than using youtube-dl.
